src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,33 @@
+@import "../../style/mixin.scss";
+@import "../../style/variables.scss";
+.introduce {
+  @include flex;
+  border: 1px solid $blackColor;
+  width: 150px;
+  height: 150px;
+  display: flex;
+  margin: 300px auto;
+
+  .image {
+    width: 100px;
+    height: 100px;
+    background-image: url("./profile.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin-top: 10px;
+  }
+  .input {
+    width: 100px;
+  }
+  .link {
+    font-size: 13px;
+  }
```

```suggestion
  }
  
  .input {
    width: 100px;
  }
  
  .link {
    font-size: 13px;
  }
```

전반적으로 nesting 잘 구현해주셨네요!
한가지 피드백 드리면 스타일 끼리는 가독성을 위해서 한칸씩 띄워주세요!